packages/api/endpoints/granules.js

```diff
@@ -724,15 +724,26 @@ async function bulkPatchGranuleCollection(req, res) {
   const granules = req.body.apiGranules;
   const granuleIds = granules.map((granule) => granule.granuleId);
   const newCollectionId = req.body.collectionId;
-  const collection = await collectionPgModel.get(
-    knex,
-    deconstructCollectionId(newCollectionId)
-  );
+  let collection;
+
+  try {
+    collection = await collectionPgModel.get(
+      knex,
+      deconstructCollectionId(newCollectionId)
+    );
+  } catch (error) {
+    if (error.message.includes('invalid collectionId')) {
```

Comment:
Is `invalid collectionId` referring to the message thrown from `deconstructCollectionId` if it ... can't? 

Does that mean the collection doesn't exist, or just that the entry is malformed? 

What does .get return if the deconstructed collection id works, but there's no collection?